Patches for the terraphast component of RAxML-NG.
* Fixes a problem with an implicit type conversion to a lesser type in a header file
Author: Kurt Lust, UAntwerpen
diff -ru raxml-ng/libs/terraphast/lib/bits.hpp.orig raxml-ng/libs/terraphast/lib/bits.hpp
--- raxml-ng/libs/terraphast/lib/bits.hpp.orig	2020-04-10 10:10:23.000000000 +0200
+++ raxml-ng/libs/terraphast/lib/bits.hpp	2020-04-10 14:33:31.000000000 +0200
@@ -11,7 +11,7 @@
 
 inline index block_index(index i) { return i / word_bits; }
 inline index base_index(index block) { return block * word_bits; }
-inline uint8_t shift_index(index i) { return i % word_bits; }
+inline index shift_index(index i) { return i % word_bits; }
 inline index set_mask(index i) { return 1ull << (i & (word_bits - 1)); }
 inline index clear_mask(index i) { return ~set_mask(i); }
 inline index prefix_mask(index i) { return set_mask(i) - 1; }
